rpc/lib/server/handlers.go

```diff
@@ -118,6 +118,8 @@ func makeJSONRPCHandler(funcMap map[string]*RPCFunc, logger log.Logger) http.Han
 		// The Server MUST NOT reply to a Notification, including those that are within a batch request.
```

Comment:
why don't you want to update this comment? instead of writing a new one

Reply:
I hadn't even seen that, which is what prompted me to firstly erraneously return a response and then later update the comment since with a notification id=="". Thank you for raising that. I'll update the comment up there.